charts/prerequisites/values.yaml

```diff
@@ -65,6 +65,34 @@ postgresql:
     # For better security, add postgresql-secrets k8s secret with postgres-password, replication-password and password
     existingSecret: postgresql-secrets
 
+gcloud-sqlproxy:
```

Comment:
It is worth adding a comment saying that using `gcloud-sqlproxy` requires the nodes in a GKE cluster to have the Cloud SQL Admin scope. In addition, the service account (default compute service account if not overrode) needs to have Cloud SQL Client role.